Admin/AbstractAdmin.php

```diff
@@ -2838,22 +2838,12 @@ public function getDashboardActions()
     }
 
     /**
-     * Setting to true will enable mosaic button for
-     * the admin screen.
-     * Setting to false will hide mosaic button for
+     * Setting to true will enable mosaic button for the admin screen. Setting to false will hide mosaic button for
      * the admin screen.
      */
-    public function setIsShowMosaicButton($isShow)
+    public function setIsShowMosaicButton($isShown)
```

Comment:
>  In fact, I think 3 methods are not necessary.

I don't agree with you. If somebody wants to hide `mosaic' button only if some condition equal true, in ListMapper - he can call method `hideMosaicButton`. I see in this solution two advantages:
1. We hide realization and `listModes` variable name.
2. IDE can highlight this method unlike `unset($this->listModes['mosaic'])`